software/controller/lib/core/v03system.cpp

```diff
@@ -197,17 +205,39 @@ void V03System::init_subsystems() {
   i2c1_.Initialize(I2C::Speed::Fast, GPIO::Port::B, /*scl_pin=*/8, /*sda_pin=*/9,
                    GPIO::AlternativeFunction::AF4);
 
+  // The following pins are used to talk to the stepper drivers on the SPI daisy chain:
```

Comment:
Aren't some of these comments redundant with the in-line ones in the calls below?